contracts/SimpleInterest.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+library SimpleInterest {
```

Comment:
It's a generalised approach of calculating the simple interest with as few operations as possible. Since it isn't coupled with the gateway as suggested, it does not necessarily have to do with SEUR/TST.

Reply:
but there aren't any examples where it's used to calculate interest. it's only used to convert sEURO to TST as far as i can see

Reply:
I should have named it `SimpleRate`. Yes, it's only used for conversion between these two tokens as of now.